package.json

```diff
@@ -96,7 +98,7 @@
     "react-resize-detector": "^4.2.0",
     "react-router": "^5.0.1",
     "react-router-dom": "^5.0.1",
-    "react-viewerbase": "0.10.0",
+    "react-viewerbase": "file:.yalc/react-viewerbase",
```

Comment:
Leftover yalc..

Reply:
@swederik it's already  updated, check the newer version:

https://github.com/OHIF/Viewers/blob/a25f5bfa6381c836f9d4cca5c2a4738248fcb81d/package.json#L101